### ISSUE TITLE
Apply ANts on 3D segmentation

### ISSUE THREAD

**Issue by liamheng** [ntustison/BRATS2013#1]
Hi, Nick

I'm doing 3D segmentation with ANTs. And I figured out the example code cannot be used for 3D segmentation. 

For my understanding, 
I shall perform the scripts createTruthLabels.pl, createNormalizedImagesForCohort.pl,  createFeatureImagesForCohort.pl, runLeaveOneOutCrossValidation.pl, createRandomForestModel.pl, applyTumorSegmentationModelForCohort.pl, refineTumorSegmentationResultsForCohort.pl one by one to finish the segmentation. Am I right?

And the script can be run by command: createTruthLabels.pl <outputDir> <cohort>. What shall be the input of <cohort> ?

Thank you so much.

Heng

**Comment by ntustison** [comment]
Those scripts are certainly available to use but they weren't meant for public consumption in that I wouldn't be able to walk you through all those scripts I used to do the BRATS competition.  That's the reason I put together the simple 2-D example that you should be able to take an apply to your 3-D data.

**Comment by liamheng** [comment]
I tried to use the 2-D example first, and it can be applied on 3-D images. However, the segmentation results are not good. I think this is because you did some preprocessing before the example, such as you added three labels of tissue to the groundtrueth. And this step seems important to the following segmentation.

**Comment by ntustison** [comment]
Yes, you need to add labels for the normal tissues: gray matter, white matter, and csf.  But you're going to need to figure that out on your own.  The scripts are pretty clean so you should be able to figure out what they're doing.
